test/SymbolGtest.cpp

```diff
@@ -0,0 +1,105 @@
+////////////////////////////////////////////////////////////////////////////
+/**
+ *  @file   SymbolGtest.cpp
+ *  @date   Wed February 23 2022
+ *  @brief  Unit test suite for the Sample class
+ */
+////////////////////////////////////////////////////////////////////////////
+
+#include <limits>
+#include <sstream>
+
+#include<gtest/gtest.h>
+#include<gmock/gmock.h>
+#include "../inc/Symbol.h"
+
+using namespace testing;
+
+class SymbolFixture: public ::testing::Test
+{
+    protected:
+        calculator::Symbol<int> intSymbolA;
+        calculator::Symbol<int> intSymbolBBound;
+        calculator::Symbol<int> intSymbolCBound;
+        calculator::Symbol<int> intSymbolDBound;
+        calculator::Symbol<int> intSymbolEBound;
+        calculator::Symbol<int> intSymbolFBound;
+        calculator::Symbol<int> intSymbolZ;
+        
+        calculator::Symbol<double> doubleSymbolA;
+        calculator::Symbol<double> doubleSymbolBBound;
+        calculator::Symbol<double> doubleSymbolCBound;
+        calculator::Symbol<double> doubleSymbolDBound;
+        calculator::Symbol<double> doubleSymbolEBound;
+        calculator::Symbol<double> doubleSymbolFBound;
+        calculator::Symbol<double> doubleSymbolZ;
+
+        SymbolFixture(): 
```

Comment:
constructors in test fixtures should only be used when you absolutely need to. Otherwise you should override the SetUp() and the TearDown() for construction and deconstruction.

https://google.github.io/googletest/primer.html#same-data-multiple-tests

Reply:
Also, only put data in the fixture that will be used for ALL or at least most tests, otherwise define the values in the test itself.

Reply:
Also also :), Fixture Names (you can think of them as a test suite also) should have the name that states the context or the test names. For complicated classes this might be something like. `GivenANewWidget` then you would inherit from that fixture and extend into `GivenAnIntializedWidget` then `GivenAWidgetWithMockValues`